tests/test.js

```diff
@@ -20,11 +20,12 @@ describe('when parsing an Ogg Stream', function(){
             .on('data', function(t){
                 tags[t.type] = t.value;
             })
-            .on('end', function(){
+            .on('finish', function(){
```

Comment:
thanks for the update, i'd like to switch these over but, you know, who has the time 🕐 , someday!